Lib/defconQt/fontView.py

```diff
@@ -1740,11 +1763,11 @@ def __init__(self, parent=None):
     def addItem(self):
 
         def mangleNewName():
-            name = "New"
+            name = self.tr("New")
             index = 0
             while self.markColorWidget.findItems(name, Qt.MatchExactly, 1):
                 index += 1
-                name = "New ({})".format(index)
+                name = name + " ({})".format(index)
```

Comment:
The concentrated strings here are bad for l10n, this needs to be an explicit list of complete `New foo` strings.